tests/bulk_create/models.py

```diff
@@ -76,11 +81,11 @@ class NullableFields(models.Model):
     # Fields not required in BulkInsertMapper
     char_field = models.CharField(null=True, max_length=4, default='char')
     email_field = models.EmailField(null=True, default='user@example.com')
-    duration_field = models.DurationField(null=True, default=datetime.timedelta(1))
```

Comment:
This redefines `duration_field` from line 68.
Not sure if it's ok to remove it here or should I create a new commit/PR.